Dockerfile

```diff
@@ -1,36 +1,33 @@
-FROM node:16 AS node-build
+FROM registry.suse.com/bci/nodejs:14 AS node-build
 WORKDIR /build
 # skip the hack/get_version_from_git.sh execution in the frontend build
 ENV VERSION=""
 # first we only add what's needed to run the makefile
 ADD Makefile /build/
 # then we add what's needed just to install node packages so that dependencies can be cached in a dedicate layer
 ADD web/frontend/package.json web/frontend/package-lock.json /build/web/frontend/
-RUN make web-deps
+RUN zypper -n in make && make web-deps
 # only as last step we add the web frontend sources, this way changing these doesn't retrigger the slow npm install
 ADD web/frontend /build/web/frontend/
 RUN make web-assets
 
-FROM golang:1.16 as go-build
+FROM registry.suse.com/bci/golang:1.16 as go-build
 WORKDIR /build
 # we add what's needed to download go modules so that dependencies can be cached in a dedicate layer
 ADD go.mod go.sum /build/
 RUN go mod download
 ADD . /build
 COPY --from=node-build /build /build
-RUN make build
+RUN zypper -n in make git && make build
```

Comment:
most likely installing `git-core` should be good enough, not the much larger `git`